lte/gateway/c/session_manager/test/test_set_session_manager_handler.cpp

```diff
@@ -192,12 +181,24 @@ TEST_F(SessionManagerHandlerTest, test_SetAmfSessionContext) {
       &server_context, &request,
       [this](grpc::Status status, SmContextVoid Void) {});
 
+  assert_sm_session_context(&request);
+  EXPECT_EQ(request.mutable_common_context()->sid().id(), IMSI1);
```

Comment:
why is this checked outside `assert_sm_session_context`

Reply:
for imsi we were checking outside `assert_sm_session_context` as in some places we were using IMSI1, some places we were using IMSI2. so to pass those test cases I moved out imis checking out of `assert_sm_session_context` function.